site/en/tutorials/curl_quickstart.ipynb

```diff
@@ -0,0 +1,355 @@
+{
```

Comment:
<p>"try out the API" -&gt; "try out the PaLM API" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done